tests/integration/test_vulnerability_detector/test_scan_results/data/configuration_template/scan_result_alert_vulnerability_removal.yaml

```diff
@@ -0,0 +1,60 @@
+- sections:
+    - section: vulnerability-detector
+      elements:
+        - enabled:
+            value: 'yes'
+        - interval:
+            value: "5s"
```

Comment:
```suggestion
            value: '5s'
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2565/commits/50cd25f78e370246762e75c1eb6610a843281094